interfaces/opensearch_client/v0/README.md

```diff
@@ -64,8 +67,8 @@ Provider provides credentials, endpoint addresses, TLS info and index-specific f
     related-endpoint: opensearch-app-consumer
     application-data:
       endpoints: 10.180.162.200:9200,10.180.162.75:9200
-      password: Dy0k2UTfyNt2B13cfe412K7YGs07S4U7
-      username: opensearch-client_4_user
+      secret-user: secret://59060ecc-0495-4a80-8006-5f1fc13fd783/cjqub6vubg2s77p3nio0
+      secret-tls: secret://59060ecc-0495-4a80-8006-5f1fc13fd783/cjqub7fubg2s77p3niog
```

Comment:
Since it's optional (and not part of the initial Mermaid diagram) we may not want to list tls here.

Reply:
please keep it as it is